docs/content/ChoiceInputField.mdx

```diff
@@ -0,0 +1,164 @@
+---
+title: ChoiceInputField
+status: Alpha
+description: The ChoiceInputField component is used to render a labelled checkbox or radio input and, optionally, associated hint text and graphic.
+source: https://github.com/primer/react/blob/main/src/ChoiceInputField.tsx
+storybook: '/react/storybook?path=/story/forms-choiceinputfield--checkbox-input-field'
+---
+
+<!--
+TODO: add `validationStatus` examples when we have the validation styling done
+-->
+
+import {ChoiceInputField, Checkbox, Radio} from '@primer/react'
+import {MarkGithubIcon} from '@primer/octicons-react'
+import {ComponentChecklist} from '../src/component-checklist'
+import {PropsTable} from '../src/props-table'
```

Comment:
Some of these imports may not be needed anymore

Reply:
I'll see what I can get rid of. When I excluded the `PropsTable` import, it caused a deployment error in Vercel.